src/StudyViewer/ExportToWorkspace.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Toaster from '@gen3/ui-component/dist/components/Toaster';
+import Button from '@gen3/ui-component/dist/components/Button';
+import { manifestServiceApiPath } from '../localconf';
+import { fetchWithCreds } from '../actions';
+
+class ExportToWorkspace extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      toasterOpen: false,
+      toasterHeadline: '',
+      exportPFBToWorkspaceGUID: '',
+      exportPFBToWorkspaceStatus: null,
+      jobFailed: false,
+    };
+  }
+
+  componentDidUpdate(prevProps) {
+    if (this.props.exportToWorkspaceAction
+      && this.props.exportToWorkspaceAction.accessibleValidationValue
+      && !this.props.exportingPFBToWorkspace
+      && this.props.exportToWorkspaceAction.accessibleValidationValue !== prevProps.exportToWorkspaceAction.accessibleValidationValue) {
+      this.exportToWorkspace(this.props.exportToWorkspaceAction);
+    }
+    if (this.props.job && !this.props.job.status && !this.state.jobFailed) {
+      this.onJobFailed('There was an error please refresh the page and try again');
+    }
+    if (this.props.job && this.props.job.status === 'Failed' && prevProps.job && prevProps.job.status !== 'Failed' && !this.state.jobFailed) {
+      this.onJobFailed('There was an error exporting your cohort.');
```

Comment:
does sower return any error details, or just "failed"?

Reply:
no error details but i will add to the message to refresh and try again, looks like i can get this error if my session goes stale while the export is in progress